src/app/rosetta/lib/block.ml

```diff
@@ -564,7 +569,8 @@ module Specific = struct
                     { Transaction.transaction_identifier=
                         {Transaction_identifier.hash= info.hash}
                     ; operations=
-                        Internal_command_info.to_operations ~coinbase info
+                        Internal_command_info.to_operations
+                          ~coinbase_receiver:block_info.creator ~coinbase info
```

Comment:
We need to add another field `coinbase_receiver` to the `blocks` table and use that here instead of the creator. Creator can be different from the receiver and all the rewards go to the receiver. Coinbase transactions have `coinbase_receiver` in the receiver field so we did not need this before.

Reply:
We can extract it from the coinbase internal command actually -- good call I forgot about coinbase elsewhere